### DISCUSSION TITLE
Shadcn UI / Radix UI example wanted

### DISCUSSION THREAD

**Discussion by edmundhung** [edmundhung/conform#262] [Announcements]
It would be great to have examples integrating Conform with components from [Shadcn UI](https://ui.shadcn.com/) and [Radix UI](https://www.radix-ui.com/). The format could be similar to the [Headless UI example](https://conform.guide/examples/headless-ui).

**Comment by anderson-chong** [comment]
Hi, is there any specific requirements for Radix UI? Is it okay if it's...
1. Created with create-react-app?
2. Using CSS instead of TailwindCSS? (Since the examples in the official documentation are styled with CSS, it's easier to just copy the styles)

**Reply by edmundhung** [reply]
Thanks for the offer!

> 1. Created with create-react-app?

I want to migrate all the non remix examples to a `vite` setup. But it's also fine to use `create-react-app` first and I can migrate all of them together.

> 2. Using CSS instead of TailwindCSS? (Since the examples in the official documentation are styled with CSS, it's easier to just copy the styles)

I am fine with both.

**Reply by balzdur** [reply]
FYI, on the Radix UI doc, you can toggle for Tailwind in the top right corner if you want (in case you prefer to use Tailwind everywhere to ease the setup + copy paste)

**Reply by anderson-chong** [reply]
@balzdur I didn't notice that, thanks! I would much prefer tailwind over CSS

**Comment by marilari88** [comment]
Hi @edmundhung I've submitted a PR #405  that includes examples of integrating Conform with components from Radix UI. 

Interestingly, while I was working on the PR, I noticed that v1 was already released, seemingly anticipating my efforts 😄 Anyway, the upgrade to v1 was straightforward, and I appreciate the smooth transition. Kudos to you and the community for the great work on the v1 changes – a significant step forward!

Looking forward to any feedback or suggestions on the PR.

**Comment by marilari88** [comment]
@edmundhung I've also included an example of Shadcn UI in the Pull Request https://github.com/edmundhung/conform/pull/489. Feel free to review it and let me know if there's anything you'd like me to add or modify.
